core-services/image-mirroring/openshift/mapping_origin_4_14

```diff
@@ -294,6 +294,7 @@ registry.ci.openshift.org/origin/4.14:service-ca-operator quay.io/openshift/orig
 registry.ci.openshift.org/origin/4.14:service-catalog quay.io/openshift/origin-service-catalog:4.14 quay.io/openshift/origin-service-catalog:4.14.0 quay.io/openshift/origin-service-catalog:latest
 registry.ci.openshift.org/origin/4.14:special-resource-operator quay.io/openshift/origin-special-resource-operator:4.14 quay.io/openshift/origin-special-resource-operator:4.14.0 quay.io/openshift/origin-special-resource-operator:latest
 registry.ci.openshift.org/origin/4.14:special-resource-rhel8-operator quay.io/openshift/origin-special-resource-rhel8-operator:4.14 quay.io/openshift/origin-special-resource-rhel8-operator:4.14.0 quay.io/openshift/origin-special-resource-rhel8-operator:latest
+registry.ci.openshift.org/origin/4.14:src-kubectl quay.io/openshift/origin-src-kubectl:4.14 quay.io/openshift/origin-src-kubectl:4.14.0 quay.io/openshift/origin-src-kubectl:latest
```

Comment:
I am 99% sure that this is not the image that we want to mirror, we only need it for running the tests.

Reply:
Probably my understanding was wrong here, I thought this step was required for using the image in tests.
It was generated by the command `make openshift-image-mirror-mappings`